docs/how-to/understanding-break-the-monolith.md

```diff
@@ -0,0 +1,46 @@
+---
+id: understanding-break-the-monolith
+title: Understanding Break The Monolith
+sidebar_label: Understanding Break The Monolith
+slug: /understanding-break-the-monolith
+---
+
```

Comment:
@dericksozo - Can you please explain what this page adds compared to the "break-the-monolith" page? I think we need to have a more technical page that explains in more detail how we handle the relations between entities (1-1, 1 to many, many to many? etc.) when these entities are located in different services as part of the BTM process.
WDYT?